src/cascadia/TerminalApp/App.idl

```diff
@@ -22,8 +22,9 @@ namespace TerminalApp
 
         Windows.Foundation.Point GetLaunchDimensions(UInt32 dpi);
         Boolean GetShowTabsInTitlebar();
-
+		
         event Microsoft.Terminal.TerminalControl.TitleChangedEventArgs TitleChanged;
+		event CloseWindowNoTabsEventArgs CloseWindow;

```

Comment:
I'd be more inclined to call this `LastTabClosed` or even something more generic like `ApplicationClosing`; @zadjii-msft, thoughts?

Reply:
An event name should describe the "thing that is triggering the event" -- like TitleChanged, above, fits into a sentence:

"The `TitleChanged`, so I want to `[]{ do this thing; }`"